content/python/concepts/strings/terms/rpartition/rpartition.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: '.rpartition()' 
+Description: 'It is a string method that is used to split a string into three parts based on a specified separator.'
+Subjects: 
+  - 'Data Science'
+  - 'Web Development'
+  - 'Computer Science'
+Tags: 
+  - 'Strings'
+  - 'Methods'
+  - 'Formatting'
+  - 'Tuples'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The **`rpartition()`** method is a string method that is used to split a string into three parts based on a specified separator. It searches for the last occurrence of the specified separator in the string and divides the string into three parts: the part before the separator, the separator itself, and the part after the separator. The method returns a tuple containing these three parts.
+
+## Syntax
+
+```pseudo
+str.rpartition(separator)
+```
+
+The `.rpartition()` method takes the following optional parameters:
+
+- `str`: This is the string on which you want to apply the `rpartition()` method.
+
+- `separator`: This is the delimiter that determines where to split the string.
```

Comment:
```suggestion
- `str`: This is the string on which you want to apply the `.rpartition()` method.
- `separator`: This is the delimiter that determines where to split the string.
```
The deleted sentence was potentially confusing because `str` is not a prameter.